skore/tests/unit/sklearn/test_find_estimators.py

```diff
@@ -19,7 +19,7 @@ def test_find_estimators():
 
 
 def test_find_estimators_with_skrub():
-    pip = skrub.tabular_learner("classification")
+    pip = skrub.tabular_pipeline("classification")
```

Comment:
I think that we should make an effort to support skrub version 0.5 and 0.6 for the moment. The one way to handle this deprecation would be to add a `tabular_pipeline` in the `skore/_externals/_skrub_compat.py` where in this file it will call `tabular_learner` if the version is `<0.6` and `tabular_pipeline` otherwise.

Then, we will never need to change code internally.

Reply:
Is that important to be compatible with both versions in the tests? As it's not in the usage of skore?  

If so, I'll go back to tabular_learner, it's still working, there is only a deprecation warning for now.